javascripts/projectsService.js

```diff
@@ -132,27 +133,27 @@ define(['underscore'], _ => {
     return _.flatten(results, (arr1, arr2) => arr1.append(arr2));
   };
 
-  var TagBuilder = function() {
-    var _tagsMap = {},
+  const TagBuilder = function() {
+    const _tagsMap = {},
       _orderedTagsMap = null;
 
     this.addTag = function(tag, projectName) {
-      var tagLowerCase = tag.toLowerCase();
+      const tagLowerCase = tag.toLowerCase();
       if (!_.has(_tagsMap, tagLowerCase)) {
         _tagsMap[tagLowerCase] = {
           name: tag,
           frequency: 0,
           projects: [],
         };
       }
-      var _entry = _tagsMap[tagLowerCase];
+      const _entry = _tagsMap[tagLowerCase];
       _entry.frequency++;
       _entry.projects.push(projectName);
     };
 
     this.getTagsMap = function() {
       // https://stackoverflow.com/questions/16426774/underscore-sortby-based-on-multiple-attributes
-      return (_orderedTagsMap =
+      return (const_orderedTagsMap =
```

Comment:
This typo needs to be undone

```suggestion
      return (_orderedTagsMap =
```